mino/router/mod.go

```diff
@@ -67,7 +67,7 @@ type Router interface {
 	GetHandshakeFactory() HandshakeFactory
 
 	// New creates a new routing table that will forward packets to the players.
```

Comment:
We should then update the comment to mention what the second parameter refers to, or use named parameters:

`New(players mino.Players, me mino.Address)`